section2/exercises/if-statements.js

```diff
@@ -44,6 +44,13 @@ Log to the console the appropriate age range
 var dogAge = 3;
```

Comment:
Good work throughout the file AND good use of indentation, white space, all those style/syntax things